### PR TITLE
Some updates after last translation

### PR DESCRIPTION
**Pull request by kuenishi**

### CODE REVIEW

File: RELEASE-NOTES.ja.md [hunk 1]
```diff
@@ -57,9 +57,14 @@
...
-  `riak_cs_console:cleanup_orphan_multipart/0` または
-  `riak_cs_console:cleanup_orphan_multipart/1` を実行します。
+- すべてのバケットを正常な状態にするため、 `riak-cs-admin
+  cleanup-orphan-multipart` を実行します。 `2014-07-30T11:09:30.000Z`
+  のような、 ISO 8601 形式の日付を引数として指定するとよいでしょう。た
+  とえば、全てのCSのアップグレードが完了した時間を指定しておくと、その
+  タイムスタンプよりも新しく作成されたマルチパートのアップロードを削除
+  しません。これによって、バケットの作成や削除、マルチパートアップロー
+  ドの開始、不正なマルチパートアップロードの削除が競合したときなどにい
+  くつかのコーナーケースがあります。
```
Comment (shino) [inline RELEASE-NOTES.ja.md]: s/あります/避けられます/ ?

**Comment by shino** [comment]
I will +1 after the above comment is addressed.

**Comment by kuenishi** [comment]
This should be rebased after #943 merged.

**Comment by kuenishi** [comment]
Rebased and force-pushed.

File: RELEASE-NOTES.ja.md [hunk 2]
```diff
@@ -110,6 +116,20 @@ GC を実行しない CS ノードは、猶予期間が正しく動作するた
...
+
+* コピーを実行中にクライアントが次のリクエストを送信するとコピーは中断
+  されます。これはクライアントの切断を検出してコピーを中止する機構の副
+  作用です。詳しくは [#932](https://github.com/basho/riak_cs/pull/932)
+  をご覧ください。
+
+* OOSインターフェースでのコピーはサポートされていません。
+
+* Multibag はオブジェクトのマニフェストとブロックを複数の異なるクラスタ
+  に分けて格納する機能です。これは Riak CS EE の機能として追加されまし
```
Comment (shino) [inline RELEASE-NOTES.ja.md]: s/EE/Enterprise/

File: RELEASE-NOTES.ja.md [hunk 3]
```diff
@@ -36,7 +36,7 @@
 * マルチパートでアップロードされたオブジェクトの ETag 不正を修正 [riak_cs/#855](https://github.com/basho/riak_cs/issues/855)
 * PUT Bucket Policy のポリシーバージョン確認の不具合を修正[riak_cs/#911](https://github.com/basho/riak_cs/issues/911)
 * コマンド成功時に終了コード 0 を返すよう修正 [riak_cs/#908](https://github.com/basho/riak_cs/issues/908)
-
+* `{error, disconnected}` が内部で notfound に書き換えられる問題を修正 [riak_cs/#929](https://github.com/basho/riak_cs/issues/929)
 
 ## アップグレードに関する注意事項
```
Comment (shino) [inline RELEASE-NOTES.ja.md:42]: The section "Riak Version" is lacking.
https://github.com/basho/riak_cs/pull/943/files#diff-89f3cba2993eebd391f32c27f3772d7bR43

**Comment by shino** [comment]
Almost good :smile:  Two comments added.

**Comment by kuenishi** [comment]
Updated.

File: RELEASE-NOTES.ja.md [hunk 4]
```diff
@@ -36,10 +36,16 @@
...
 
+### Riak Version
+
+ここリリースは Riak 1.4.10 上でテストされました。
```
Comment (shino) [inline RELEASE-NOTES.ja.md]: s/ここ/この/

**Comment by shino** [comment]
+1 other than just tiny typo.

**Comment by kuenishi** [comment]
+1 8f10d86 r=shino

**Comment by kuenishi** [comment]
@borshop merge

**Pull request merged**